src/libsyncengine/progress/progressinfo.cpp

```diff
@@ -140,7 +142,9 @@ bool ProgressInfo::setProgressComplete(const SyncPath &path, const SyncFileStatu
 
     const auto it = _currentItems.find(normalizedPath);
```

Comment:
The two blocks looks identical.  Refactoring to remove redundancy would let us write a fix in a single place.